src/compiler/typer.cc

```diff
@@ -143,6 +146,19 @@ class LazyTypeCache FINAL : public ZoneObject {
         return Type::Function(Get(kFloat32x4), Get(kFloat32x4),
                               Type::Integral32(), Type::Integral32(),
                               Type::Integral32(), Type::Integral32(), zone());
+      // Int32x4
+      case kInt32x4:
+        return CreateInt32x4();
+      case kInt32x4Tagged:
+        return CreateInt32x4Tagged();
+      case kInt32x4Func2:
+        return Type::Function(Get(kInt32x4), Get(kInt32x4), Get(kInt32x4),
+                              zone());
+      case kInt32x4Func2n:
```

Comment:
kInt32x4Fun2n => Int32x4 func(number1, number2)
So change it to kInt32x4Fun4i ==> Int32x4 func(int32, int32, int32, int32)